src/settings/RoutingAddress/hooks/useRoutingAddressSettings/useRoutingAddressSettings.js

```diff
@@ -22,20 +23,21 @@ export const useRoutingAddressSettings = (options = {}) => {
     isFetching,
     isLoading,
     refetch,
+    error,
   } = useQuery({
     queryKey: [namespace],
     queryFn: async () => {
-      const response = await ky.get(ORDERS_STORAGE_SETTINGS_API, { searchParams }).json();
+      const response = await ky.get(ORDERS_STORAGE_SETTINGS_API, { searchParams }).json().catch(onError);
```

Comment:
why do you need catch here when onError is option of useQuery and it can handle errors inside `queryFn`